charmdir.go

```diff
@@ -466,16 +464,25 @@ func (v *vcsCMD) commonErrHandler(err error, charmPath string) error {
 }
 
 // The first check checks for the easy case of the current charmdir has a git folder.
-// There can be cases when the charmdir actually uses git and is just a subdir, thus the below check
+// There can be cases when the charmdir actually uses git and is just a subdir, hence the below check
 func usesGit(charmPath string) bool {
 	if _, err := os.Stat(filepath.Join(charmPath, ".git")); err == nil {
 		return true
 	}
+	ctx, cancel := context.WithTimeout(context.Background(), 2*time.Second)
```

Comment:
Can we make `2*time.Second` a constant.

`CmdWaitTime` or similar.

Reply:
can do :+1: